ui/util/url.js

```diff
@@ -70,3 +70,41 @@ exports.generateInitialUrl = hash => {
   }
   return url;
 };
+
+exports.generateLbryUrl = (canonicalUrl, permanentUrl) => {
```

Comment:
This isn't really a lbry url. Maybe it should be called something like `generateLbryContentUrl`?